doc/README.md

```diff
@@ -34,6 +34,7 @@ Work in progress.
 
 * [Quick install](https://github.com/jitsi/jitsi-meet/blob/master/doc/quick-install.md)
 * [Docker install](https://github.com/jitsi/docker-jitsi-meet/blob/master/README.md)
+* [Ansible install](https://github.com/tderflinger/jitsi-deployment)
```

Comment:
I assume that creating an entry on Wiki and add a link to your blog article instead would be appreciated instead since this link goes 404 when your repo would be deleted.

Reply:
That article could include another instruction on Ansible installation, which was indicated here: https://github.com/jitsi/jitsi-meet/pull/6572#issuecomment-624640833

Reply:
My repo on GitHub won't be deleted and that's where all the files are located. It is secure. Yes, we could include the alternative Ansible roles also in the Wiki page.